javascript/chronometer.js

```diff
@@ -1,26 +1,72 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    this.current_seconds = 0;
+    this.interval_seconds = 0;
+
+    this.current_milliseconds = 0;
+    this.interval_milliseconds = 0;
   }
-  startClick(callback) {
-    // ... your code goes here
+
+  startTimer(callback) {
+    this.startMilliseconds(callback);
+    this.startSeconds(callback);
   }
+
+  startSeconds(callback) {
+    this.interval_seconds = setInterval(() => {
+      this.current_seconds++;
+      callback();
+    }, 1000);
+  }
+
+  startMilliseconds(callback) {
+    this.interval_milliseconds = setInterval(() => {
+      this.current_milliseconds++;
+      callback();
+    }, 1);
+  }
+
   getMinutes() {
-    // ... your code goes here
+    return Math.floor(this.current_seconds / 60);
   }
+
   getSeconds() {
-    // ... your code goes here
+    return this.current_seconds - this.getMinutes() * 60;
+  }
+
+  getMilliseconds() {
+    return this.current_milliseconds % 100;
   }
-  twoDigitsNumber() {
-    // ... your code goes here
+
+  twoDigitsNumber(current_time) {
+    if (current_time < 10) {
+      return "0" + String(current_time);
+    } else {
+      return String(current_time);
+    }
   }
+
   stopClick() {
-    // ... your code goes here
+    clearInterval(this.interval_milliseconds);
+    clearInterval(this.interval_seconds);
   }
+
   resetClick() {
-    // ... your code goes here
+    this.current_milliseconds = 0;
+    this.current_seconds = 0;
   }
+
   splitClick() {
-    // ... your code goes here
+    const minutes = this.getMinutes();
+    const seconds = this.getSeconds();
+    const milliseconds = this.getMilliseconds();
+
+    if ((minutes < 10 || seconds < 10) && milliseconds > 9) {
+      return `${0}${minutes}:${0}${seconds}:${milliseconds}`;
+    } else if ((minutes < 10 || seconds < 10) && milliseconds < 9) {
+      return `${0}${minutes}:${0}${seconds}:${0}${milliseconds}`;
+    } else {
+      return `${minutes}:${seconds}:${milliseconds}`;
+    }
```

Comment:
the conditional above works good but you already have a method that converts the number into a two digit string, so this code is a little redundant. Instead you could have use that.